src/platform/plugins/shared/kibana_react/public/markdown/markdownStyles.ts

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { css } from '@emotion/react';
+import { transparentize, useEuiTheme } from '@elastic/eui';
+
+// Default styles for Markdown element
+//
+// 1. Links
+// 2. Headings
+// 3. Images
+// 4. Blockquotes
+// 5. Horizontal rules
+// 6. Lists
+// 7. Tables
+// 8. Code blocks
+
+export const markdownStyles = (isReversed: boolean) => {
+  const { euiTheme } = useEuiTheme();
```

Comment:
I know it seems this works, but could we instead pass the value of `euiTheme` from an invocation of `useEuiTheme` from a component.